contracts/Tokens/Prime/Prime.sol

```diff
@@ -151,6 +145,7 @@ contract Prime is IPrime, AccessControlledV8, PausableUpgradeable, MaxLoopsLimit
      * @param _stakingPeriod total number of seconds for which user needs to stake to claim prime token
      * @param _minimumStakedXVS minimum amount of XVS user needs to stake to become a prime member (scaled by 1e18)
      * @param _maximumXVSCap maximum XVS taken in account when calculating user score (scaled by 1e18)
+     * @param _timeBased A boolean indicating whether the contract is based on time or block.
      * @custom:error Throw InvalidAddress if any of the address is invalid
      * @custom:error Throw InvalidBlocksPerYear if blocks per year is 0
```

Comment:
```suggestion
```
I think we are not throwing these errors here, so we can remove these netspec comments.